plugins/HHAnalyzer.cc

```diff
@@ -195,6 +242,11 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             else {
                fillTriggerEfficiencies(leptons[ilep1], leptons[ilep2], dilep);
             }
+
+            if (!dilep.isOS)
+                continue;
+            if (!(leptons[ilep1].hlt_DR_matchedObject < 0.3 && leptons[ilep1].hlt_DR_matchedObject < 0.3))
+                continue;
```

Comment:
This should be done only for data, otherwise we implicitly ask some trigger requirement in MC. Also, I did put 0.3 following some basic study, I think we can go to 0.1 (I remember seeing this recommended somewhere, will check).